ext/sockets/php_sockets.h

```diff
@@ -153,31 +153,31 @@ PHP_SOCKETS_API bool socket_import_file_descriptor(PHP_SOCKET socket, php_socket
 #define PHP_NORMAL_READ 0x0001
 #define PHP_BINARY_READ 0x0002
 
-#ifdef WIN32
+#ifdef _WIN32
```

Comment:
it might work with PHP_WIN32 wdyt ?

Reply:
It will work indeed. Added in the next commit.

Reply:
Otherwise, a very extreme edge case here would be that some project includes php_socket.h file and winsock2.h file before it manually without defining the PHP_WIN32 but this is just hypothetical. Because the PHP_WIN32 is very specific PHP defined and _WIN32 is omnipresent. But no worries. If it comes to that we'll need to recheck more files anyway later on.